cmd/internal/cli/pull.go

```diff
@@ -173,7 +175,7 @@ func pullRun(cmd *cobra.Command, args []string) {
 		libexec.PullShubImage(name, args[i], force, noHTTPS)
 	case HTTPProtocol, HTTPSProtocol:
 		libexec.PullNetImage(name, args[i], force)
-	default:
+	case DockerProtocol:
```

Comment:
This should _not_ be passing CI (I think). For instance, `singularity pull oci://[...]` will fail here. The reason we were using  `default` is because anything that's not `library`, `http`/`https`, or `shub`, are all still potentially valid URI's if they are handled by image transport code (this is also what handles `docker://`). If CI is passing, it means we're not testing `singularity pull oci:` or other non-`docker://` URIs